aeecleclair/Hyperion
# Add test structure for Alembic migrations

### Description

Testing migrations will allow to:
 - make sure that our migrations are valid, and usable with sqlite
 - see if SQLAlchemy models differ from the database structure constructed by running all migrations
 - make sure we don't have diverging head

### Checklist

- [x] Extended the documentation, if necessary
